section3/exercises/loops.js

```diff
@@ -23,25 +23,63 @@ for (var i = 0; i < 3; i++) {
 }
 
 // What is logged to the console for this second example? Why?
-// EXPLAIN:
-
+// EXPLAIN: the console will log 0, 1, and 2, becasue the value of i starts at 0 incriments by 1 each time the loop completes, and will run as long as the value of i is less than three. When the value of i is equal to 3 the conditional is no longer true and the interpritor exits the loop.
 
  // YOU DO: Write code below that logs to the console, the sum of 2 plus 2, 7 times:
 
+ /*
+ Create a for loop
+ Declare a new iteration variable i limiting the scope with the keyword let
+ and assign the variable i an innitial value of 0
+ continue looping as long as the value assigned to i is less than the integer 7
+ incriment the value assigned to i after each loop iteration
+ */
+ for (let i = 0; i < 7; i++) {
```

Comment:
I see you are using `let` in a few places - this is a little tricky because I know you have past coding experience.

In Mod 1, we start by teaching mostly ES5 syntax which only has `var` as a keyword to declare variables. ES6 introduces `let` and `const`; many times people think `var` and `let` are interchangeable but they actually have differences related to scope. As you work on paried/group projects, I imagine your instructors will prefer you stick to using `var` as most classmates will not be in the place to get into the scope stuff. They may also ask you to do the same for solo work, or may be ok with `let`. Just wanted to give you a heads up!